tests/test_edgeql_select.py

```diff
@@ -6304,3 +6304,14 @@ async def test_edgeql_select_array_common_type_01(self):
         for row in res:
             self.assertEqual(row[0].__tname__, "test::User")
             self.assertEqual(row[1].__tname__, "test::Issue")
+
+    @test.xfail("""
+        SchemaDefinitionError: cannot alter object type 'std::Object':
+        module std is read-only
+    """)
```

Comment:
Mention #2380?

Reply:
Do you mean #2387 (which I added a reference to)?

I don't see how #2380 is related